app/views/admin/loans/timeline/_step.html.slim

```diff
@@ -20,9 +20,9 @@
         i.fa.fa-fw.fa-calendar
     .summary = entry.summary
 
-  td class=classes
+  td class="#{classes} step-date step-start-date" data-id=entry.id data-precedent-id=entry.schedule_parent_id
     = entry.display_start_date ? l(entry.display_start_date, format: :short) : ""
-  td class=classes
+  td class="#{classes} step-date step-end-date"  data-id=entry.id data-precedent-id=entry.schedule_parent_id
```

Comment:
Great point! I forgot about that!

That's the same reason we're reusing the `classes` thing.

This is fine as is.